device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsSessionHandler.java

```diff
@@ -362,7 +362,8 @@ boolean sendMessage(Message message)
                         return false;
                     }
                 }
-                else if (subscriptionType == DEVICE_OPERATION_TWIN_UNSUBSCRIBE_DESIRED_PROPERTIES_REQUEST)
+                else //noinspection StatementWithEmptyBody
+                    if (subscriptionType == DEVICE_OPERATION_TWIN_UNSUBSCRIBE_DESIRED_PROPERTIES_REQUEST)
```

Comment:
This if statement wraps a TODO. Wanted to check with the team if this TODO is necessary, and whether I should create a task for and remove this if block

Reply:
This TODO isn't on our task list, and is no hurry to be done. No customer has asked for our client to be able to unsubscribe from methods/twins. I left his comment here just as a helpful note about where logic like that would go if we ever pursued it

Reply:
I'm fine with whatever you think is best here, whether you remove the if block or comment it out

Reply:
Usually I have seen todos in code which were intended to do tasks but are never done by whoever added it. Good to add a task for sure if you plan to remove this.

Btw is this referenced anywhere or in other words do we ever reach here?

Reply:
I don't see it being referenced anywhere. I think I will leave it just because I don't want to create a task that we might not ever get to.